Bug: Failed unit test - test_clip_manager.py

Introduced in https://github.com/nikopueringer/CorridorKey/pull/11

```
C:\Development\repos\CorridorKey>uv run pytest --cov=CorridorKeyModule --cov-branch
================================================= test session starts =================================================
platform win32 -- Python 3.14.2, pytest-9.0.2, pluggy-1.6.0
rootdir: C:\Development\repos\CorridorKey
configfile: pyproject.toml
testpaths: tests
plugins: anyio-4.12.1, cov-7.0.0
collected 154 items

tests\test_clip_manager.py ..................F

====================================================== FAILURES =======================================================
___________________________________________ TestMapPath.test_basic_mapping ____________________________________________
tests\test_clip_manager.py:104: in test_basic_mapping
    assert result == "/mnt/ssd-storage/Projects/Shot1"
E   AssertionError: assert '/mnt/ssd-sto...rojects/Shot1' == '/mnt/ssd-sto...rojects/Shot1'
E
E     - /mnt/ssd-storage/Projects/Shot1
E     ?                 ^
E     + /mnt/ssd-storage\Projects/Shot1
E     ?                 ^
=============================================== short test summary info ===============================================
FAILED tests/test_clip_manager.py::TestMapPath::test_basic_mapping - AssertionError: assert '/mnt/ssd-sto...rojects/Shot1' == '/mnt/ssd-sto...rojects/Shot1'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================ 1 failed, 18 passed in 2.04s =============================================
```

--- a/clip_manager.py
+++ b/clip_manager.py
@@ -54,10 +54,8 @@ def map_path(win_path: str) -> str:
     if win_path.upper().startswith(WIN_DRIVE_ROOT.upper()):
         # Remove drive letter
         rel_path = win_path[len(WIN_DRIVE_ROOT) :]
-        # Flip slashes
-        rel_path = rel_path.replace("\\", "/")
-        # Combine
-        linux_path = os.path.join(LINUX_MOUNT_ROOT, rel_path)
+        # Combine and flip slashes
+        linux_path = os.path.join(LINUX_MOUNT_ROOT, rel_path).replace("\\", "/")
         return linux_path
 
     # If not on V:, maybe it's already a linux path or invalid?

